docs/supported-technologies.asciidoc

```diff
@@ -80,6 +80,8 @@ The Node.js agent will automatically instrument the following modules to give yo
 |https://www.npmjs.com/package/elasticsearch-js[elasticsearch-js] |>=8.0.0 |Will instrument all queries
 |https://www.npmjs.com/package/graphql[graphql] |>=0.7.0 <15.0.0 |Will instrument all queries
 |https://www.npmjs.com/package/handlebars[handlebars] |* |Will instrument compile and render calls
+|https://www.npmjs.com/package/jade[jade] |* |Will instrument compile and render calls
```

Comment:
Because of the lack of support for older Jade versions, we should also update this:

```suggestion
|https://www.npmjs.com/package/jade[jade] |>=0.5.6 |Will instrument compile and render calls
```